src/tiled/mapdocument.cpp

```diff
@@ -335,6 +335,19 @@ void MapDocument::resizeMap(const QSize &size, const QPoint &offset, bool remove
     // TODO: Handle layers that don't match the map size correctly
 }
 
+
+void MapDocument::autocropMap()
+{
+    // Since autocrop option is allowed only for Tile Layer
```

Comment:
Since you never know when this function may get called in the future, at this level I'm usually inclined to insert an explicit check here, like:
```c++
if (!mCurrentLayer || !mCurrentLayer->isTileLayer())
    return;
```